keps/sig-node/4438-container-restart-termination/README.md

```diff
@@ -207,13 +207,19 @@ the restart during Pod termination.
 List the specific goals of the KEP. What is it trying to achieve? How will we
 know that this has succeeded?
 -->
+The following behaviors should be maintained during pod termination:
+- sidecar containers restarting
+- startupProbe and readinessProbe probing
+- container lifecycle hooks running
 
 ### Non-Goals
 
 <!--
 What is out of scope for this KEP? Listing non-goals helps to focus discussion
 and make progress.
 -->
+The following behaviors are out of scope (and probably wrong):
+- (main) containers restarting
```

Comment:
actually, we also have to allow the main containers to restart during another container's termination.
(as a result of implementing the non-blocking container termination)

I'd add something like `make everything work during the pod termination (configMaps and secrets update?)`

(may need more discussion about this as we said in the last sidecar wg meeting)